styles.css

```diff
@@ -0,0 +1,114 @@
+body {
+  background-color: #FFD9D9;
+  margin: 60px;
+  max-width: 960;
+}
+
+* {
+  font-family: "century gothic"; 
```

Comment:
You should avoid the `*` selector when possible as it will have to run through each element of the page and attempt to apply that css rule.  It would be more appropriate to apply a default style like font to the `body`. 🔨